includes/admin_settings.php

```diff
@@ -332,7 +347,56 @@ function tsml_settings_page()
                         </form>
                     </div>
 
-                    <div class="postbox stack">
+                    <div class="postbox stack">                        
+                        <!-- Entity Form -->
+                        <?php
+                        $tsml_entity = tsml_get_entity();
+                        ?>
+                        <div class="stack compact">
+                            <h2>
+                                <?php _e('Service Entity Information', '12-step-meeting-list') ?>
+                            </h2>
+                            <p>
+                                <?php echo sprintf(__('Enter information for your service entity here to help identity the meeting source when sharing feeds with others.', '12-step-meeting-list'), get_post_type_archive_link('tsml_meeting')) ?>
```

Comment:
don't need the `sprintf` here - we'd only need that if you were adding params to the i18n string